lib/model/entity/fiscaal/SaldoCommissie.enum.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * SaldoCommissie.enum.php
+ *
+ * @author G.J.W. Oolbekkink <g.j.w.oolbekkink@gmail.com>
+ *
+ */
+abstract class SaldoCommissie implements PersistentEnum {
```

Comment:
Als eerste stap snap ik deze keuze, maar het zet een slecht precedent om dit te hardcoden, want de volgende programmeur gaat hier commissies aan toevoegen in plaats van een abstractie te maken.